CHANGELOG.md

```diff
@@ -94,6 +94,17 @@ released and the release notes may change significantly before then.
 [go]:https://golang.org
 [ktf]:https://github.com/kong/kubernetes-testing-framework
 
+## [1.3.1] - 2021/06/03
```

Comment:
Please define the link at the bottom of this file.

Reply:
:+1: